addons/mail/tests/test_rtc.py

```diff
@@ -109,7 +109,6 @@ def test_10_start_call_in_chat_should_invite_all_members_to_call(self):
         with self.assertBus(
             [
                 (self.cr.dbname, 'mail.channel', channel.id),  # update new session
-                (self.cr.dbname, 'res.partner', test_user.partner_id.id),  # mark below message as read (not asserted below)
```

Comment:
Didn't the diff break the auto mark as read then? (same for other occurrences)

Reply:
This was only required due to the way the unread message counter was comptued. It is not required anymore.